src/App.js

```diff
@@ -6,13 +6,13 @@ import Categories from './components/Categories';
 
 function App() {
```

Comment:
- [ ] Good job with the design however, kindly consider removing the white space around the main app so it looks more professional

|Your app|Zeplin design|
|-|-|
|![Screenshot_20221111_194526](https://user-images.githubusercontent.com/25227022/201410025-fd1ffff2-86f4-43d4-b39f-c279c50c1be8.png)|![Screenshot_20221110_210233](https://user-images.githubusercontent.com/25227022/201410142-78eb8f29-ff87-4daf-adf5-e2c9b6703f54.png)|